mongodbatlas/provider_test.go

```diff
@@ -121,3 +122,26 @@ func TestDecodeID(t *testing.T) {
 		t.Fatalf("Bad TestDecodeID return \n got = %#v\nwant = %#v \ndiff = %#v", got, got2, diff)
 	}
 }
+
```

Comment:
Did we do a manual test to check that Infrastructure Tool is getting set?

Reply:
Just wanted to make sure - not sure what you mean see GUI?

Reply:
Sorry, I wanted to say the DEBUG Logs, 

run the apply with debug option - `TF_LOG=DEBUG  terraform apply --auto-approve`

There you can see the label when create and read the cluster.

Reply:
Makes sense. Thank you!